public/locales/en/translation.json

```diff
@@ -968,7 +968,8 @@
       "tabs": {
         "metadata": "Meta data",
         "repeat": "Rephotography"
-      }
+      },
+      "name": "Photographie"
```

Comment:
Becarefull, this is the english `translation` file

Reply:
Thanks, must be a copy paste mistake